### PR TITLE
Update GPG rotation docs with excrutiating detail.

### PR DESCRIPTION
**Pull request by bhearsum**
I _think_ this covers everything we can write down publicly....

### CODE REVIEW

File: signing/rotate_gpg_signing_key.rst [hunk 1]
```diff
@@ -22,6 +23,98 @@ Generate a new signing subkey
...
+        sub   rsa4096 2021-05-17 [S] [expires: 2023-05-17]
+        sub   rsa4096 2023-05-05 [S] [expires: 2025-05-04]
+
+    f) base64 encode `new.key` and update the `gpgPubkey` entry in the `cloudops-infra` repo for `firefoxci-adhoc-3`::
+
+        cat new.key | base64 -w0
+
+    g) Open a Pull Request with the changes; wait for it to get merged.
+
+2) Update the GPG username and password for `firefox-ci-adhoc-3` in the relengworker SOPS repository.
```
Comment (jcristau) [inline signing/rotate_gpg_signing_key.rst:80]:
```suggestion
2) Update the GPG username and password for `firefoxci-adhoc-3` in the relengworker SOPS repository.
```

File: signing/rotate_gpg_signing_key.rst [hunk 2]
```diff
@@ -22,6 +23,98 @@ Generate a new signing subkey
...
+3) Deploy production scriptworkers to pick up the changes you made above.
+4) Open a PR with a new signing manifest in the `Adhoc Signing`_ repository that will sign with GPG (note: when it does a test signing as part of opening a PR, it will not be using the new keys, because PRs use dep certs).
```
Comment (jcristau) [inline signing/rotate_gpg_signing_key.rst:82]: Should we link to an example PR here so we can basically re-use the same manifest?
(Alternatively, if we keep the manifest around, we just have to trigger promote, no need for a new PR.)

**Review by jcristau — APPROVED** [review]
Great, thank you; just one typo and a suggestion.

File: signing/rotate_gpg_signing_key.rst (hunk 2 above)
Comment (bhearsum) [reply signing/rotate_gpg_signing_key.rst:82]: That's a good idea, let's do it!

**Pull request merged**